tests/tensorflow_autolog/test_tensorflow_autolog.py

```diff
@@ -202,3 +207,63 @@ def duplicate_autolog_tf_estimator_run():
 def test_duplicate_autolog_second_overrides(duplicate_autolog_tf_estimator_run):
     metrics = client.get_metric_history(duplicate_autolog_tf_estimator_run.info.run_id, 'loss')
     assert all((x.step - 1) % 4 == 0 for x in metrics)
+
+
+@pytest.mark.large
+def test_keras_autolog_ends_auto_created_run(random_train_data, random_one_hot_labels):
+    mlflow.tensorflow.autolog(every_n_iter=5)
+
+    data = random_train_data
+    labels = random_one_hot_labels
+
+    model = tf.keras.Sequential()
+
+    model.add(layers.Dense(64, activation='relu', input_shape=(32,)))
+    model.add(layers.Dense(64, activation='relu'))
+    model.add(layers.Dense(10, activation='softmax'))
+
+    model.compile(optimizer=tf.train.AdamOptimizer(0.001),
+                  loss='categorical_crossentropy',
+                  metrics=['accuracy'])
+
+    model.fit(data, labels, epochs=10)
+
+    assert mlflow.active_run() is None
+
+
+@pytest.mark.large
+def test_keras_autolog_persists_manually_created_run(random_train_data, random_one_hot_labels):
```

Comment:
```suggestion
def test_keras_autolog_logs_to_manually_created_run(random_train_data, random_one_hot_labels):
```

Nit: just a small suggestion around the naming of these tests (replace "persist" -> "logs_to")

Reply:
Actually, these tests only check that a run is properly deleted/persisted when the run is/isn't automatically created by `autolog`. This actually brings up another point though:  the tests dependent on the pytest fixture `tf_estimator_random_data_run` all perform the tests with manually created runs. There should be tests that test the same for runs that are automatically created - I'll work on making this update.